models/zeroex/base/zeroex_base_api_fills.sql

```diff
@@ -0,0 +1,264 @@
+                0x4aa817c6f383c8e8ae77301d18ce48efb16fd2be,
+                0x4ef40d1bf0983899892946830abf99eca2dbc5ce, 
+                -- exchange proxy
+                0xdef189deaef76e379df891899eb5a00a94cbc250
+                )
+                AND (
+                    bytearray_position(INPUT, 0x869584cd ) <> 0
+                    OR bytearray_position(INPUT, 0xfbc019a7 ) <> 0
+                )
+                
+                {% if is_incremental() %}
+                AND block_time >= date_trunc('day', now() - interval '7' day) 
+                {% endif %}
+                {% if not is_incremental() %}
+                AND block_time >= cast('{{zeroex_v3_start_date}}' as date)
+                {% endif %}
+    ) temp
+    group by tx_hash
+
+),
+
+ERC20BridgeTransfer AS (
+    SELECT 
+            logs.tx_hash,
+            logs.block_number                      AS block_number,
+            INDEX                                   AS evt_index,
+            logs.contract_address,
+            block_time                              AS block_time,
+            bytearray_substring(DATA, 142, 20) AS maker,
+            bytearray_substring(DATA, 172, 20) AS taker,
+            bytearray_substring(DATA, 14, 20) AS taker_token,
+            bytearray_substring(DATA, 45, 20) AS maker_token,
+            bytea2numeric(bytearray_substring(DATA, 77, 20)) AS taker_token_amount_raw,
+            bytea2numeric(bytearray_substring(DATA, 110, 20)) AS maker_token_amount_raw,
+            'ERC20BridgeTransfer'                   AS type,
+            zeroex_tx.affiliate_address             AS affiliate_address,
+            TRUE                                    AS swap_flag,
+            FALSE                                   AS matcha_limit_order_flag
+    FROM {{ source('base', 'logs') }} logs
+    INNER JOIN zeroex_tx ON zeroex_tx.tx_hash = logs.tx_hash
+    WHERE topic0 = 0x349fc08071558d8e3aa92dec9396e4e9f2dfecd6bb9065759d1932e7da43b8a9
+    
+    {% if is_incremental() %}
+    AND block_time >= date_trunc('day', now() - interval '7' day)
+    {% endif %}
+    {% if not is_incremental() %}
+    AND block_time >= cast('{{zeroex_v3_start_date}}' as date)
+    {% endif %}
+
+), 
+BridgeFill AS (
+    SELECT 
+            logs.tx_hash,
+            logs.block_number                      AS block_number,
+            INDEX                                           AS evt_index,
+            logs.contract_address,
+            block_time                                      AS block_time,
+            bytearray_substring(DATA, 13, 20) AS maker,
+            0xdef1c0ded9bec7f1a1670819833240f027b25eff AS taker,
+            bytearray_substring(DATA, 45, 20) AS taker_token,
+            bytearray_substring(DATA, 77, 20) AS maker_token,
+            bytea2numeric(bytearray_substring(DATA, 110, 20)) AS taker_token_amount_raw,
+            bytea2numeric(bytearray_substring(DATA, 142, 20)) AS maker_token_amount_raw,
```

Comment:
i don't think we're using `bytea2numeric` function anymore on dunesql. can you just leave the data type as-is from `DATA` source field? if not, cast to `uint256`?

Reply:
we use `uint256` in `dex.trades` lineage

Reply:
i see it used in a few spots, i will only comment here, but applies across the whole PR